tests/fixtures/marine/marine_fixtures.py

```diff
@@ -16,15 +16,27 @@ def extracted_fields_from_tcp_packet() -> List[str]:
 
 
 @pytest.fixture
-def tcp_packet() -> bytes:
+def tcp_payload() -> bytes:
+    return b"payload"
+
+
+@pytest.fixture
+def tcp_packet(tcp_payload) -> bytes:
     packet = (
         ethernet.Ethernet(src_s="00:00:00:12:34:ff", dst_s="00:00:00:ff:00:1e")
         + ip.IP(src_s="10.0.0.255", dst_s="21.53.78.255")
-        + tcp.TCP(sport=16424, dport=41799)
+        + tcp.TCP(
+            sport=16424, dport=41799, flags=0b00010000, body_bytes=tcp_payload
```

Comment:
There's some `tcp.ACK` const somewhere in pypacker, use it and remove the comment

Reply:
Done